css/main.css

```diff
@@ -0,0 +1,182 @@
+@import "fonts.css";
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fff;
+  font-family: 'imperial', Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+.nav_button {
+  border-radius: 3px;
+  color: #fff;
+  font-family: 'franklin', Arial, Helvetica, sans-serif;
+  padding: 10px;
+  background-color: #567b95;
+  border: solid #326891 1px;
+  text-decoration: none;
+  display: inline-block;
+  cursor: pointer;
+  text-align: center;
+  margin: 3px;
+  float: right;
+  font-size: 11px;
+  line-height: 11px;
+  font-weight: 700;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+}
+
+.nav_button:hover {
+  border: solid#326891 1px;
+  background: #326891;
+}
+
+.navbar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 43px;
+  width: 100%;
+  border-bottom: solid 1px rgb(100, 100, 100);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background: #fff;
+}
+
+.left_items {
+  width: 300px;
+  justify-content: space-around;
+}
+
+.left_items a {
+  margin-right: 10px;
+  color: #000;
+}
+
+.centered_items {
+  width: auto;
+}
+
+.centered_items img {
+  height: 30px;
+}
+
+.right_items {
+  width: 300px;
+  justify-items: right;
+}
+
+.nytLogo {
+  height: 40px;
+}
+
+.image_center {
+  margin: 120px auto 0 auto;
+  width: 945px;
+}
+
+.text_article {
+  width: 585px;
+  margin: 0 auto;
+  margin-top: 20px;
+}
+
+.img_full {
+  width: 100%;
+}
+
+.theory_grid {
+  width: 945px;
+  display: grid;
+  grid-template-columns: repeat(4, 236px);
+  grid-template-rows: repeat(4, 137px);
+  background-image: url(../images/theoryImage.png);
+  background-size: cover;
+}
+
+.theory_grid_item {
+  font-family: 'franklin', Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 300;
+  color: #000;
+}
+
+.theory_grid_block {
+  grid-column-start: 1;
+  grid-column-end: 5;
+}
+
+.caption {
+  color: #666;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.main-tittle {
+  font-size: 40px;
+  font-family: 'chetelman', Arial, Helvetica, sans-serif;
+  font-style: italic;
+  line-height: 48px;
+  color: #121212;
+  font-weight: 700;
+  width: 600px;
+  margin: 0 auto;
+}
+
+.text_article p {
+  font-family: 'imperial', Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 30px;
+  color: #333;
+}
+
+.social-share {
+  width: 600px;
+  height: auto;
+  margin: 25px auto 0 auto;
+}
+
+.social-icon {
+  background-color: #eee;
+  border-radius: 50%;
+  margin: 5px;
+  padding: 7px;
+  width: 30px;
+  height: 30px;
+}
+
+.footerNavLinks .leftLinks li {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+  margin-top: 0.25rem;
+}
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<li>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
.footerNavLinks .leftLinks li {

/* after */
.li-class {
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_